### ISSUE TITLE
[SC009]validate layout and description task 12 

### ISSUE THREAD

**Issue by vuthypouk** [pnc-virtual-company/2018-selection#24]
please read and understand about the interface with the requirement  some point that you want to change

**Comment by SopheakWEP** [comment]
I agree to validate this task. I'm not clear about this. But I will search more

**Comment by SopheakWEP** [comment]
On this task, I think Skeleton has the library already. I do not need to find pie chart from outside.

**Comment by nuon-neourng** [comment]
Yes I agree

**Comment by CheaChan** [comment]
Dear sir,

This task we not yet to work on the process we just work for understanding the description that client provides to us, so we move it into done because we thought we all understanding of this task.
